src/store/enhancedImageBuilderApi.ts

```diff
@@ -53,7 +53,8 @@ const enhancedApi = imageBuilderApi.enhanceEndpoints({
     },
     composeImage: {
       onQueryStarted: async (
-        { composeRequest },
+        // eslint-disable-next-line no-empty-pattern
+        {},
```

Comment:
Not super sure about this one. If we need to disable the linting on the line I'd rather disable the original warning than disabling it on a newly introduced change. What do you think?

We could probably do something like this
```suggestion
        // eslint-disable-next-line @typescript-eslint/no-unused-vars
        { composeRequest },
```

Or we could indicate that we don't care about the first argument and use
```suggestion
      onQueryStarted: async (_, { dispatch, queryFulfilled }) => {
```

which should hopefully solve both of the warning at once.